google-ads-examples/src/main/java/com/google/ads/googleads/examples/shoppingads/RejectMerchantCenterLink.java

```diff
@@ -0,0 +1,164 @@
+    } catch (GoogleAdsException gae) {
+      // GoogleAdsException is the base class for most exceptions thrown by an API request.
+      // Instances of this exception have a message and a GoogleAdsFailure that contains a
+      // collection of GoogleAdsErrors that indicate the underlying causes of the
+      // GoogleAdsException.
+      System.err.printf(
+          "Request ID %s failed due to GoogleAdsException. Underlying errors:%n",
+          gae.getRequestId());
+      int i = 0;
+      for (GoogleAdsError googleAdsError : gae.getGoogleAdsFailure().getErrorsList()) {
+        System.err.printf("  Error %d: %s%n", i++, googleAdsError);
+      }
+    }
+  }
+
+  /**
+   * Runs the example.
+   *
+   * @param googleAdsClient the Google Ads API client.
+   * @param customerId the client customer ID of the Google Ads account to reject the link request.
+   * @param merchantCenterAccountId the Merchant Center account ID for the account requesting to
+   *     link.
+   * @throws GoogleAdsException if an API request failed with one or more service errors.
+   */
+  private void runExample(
+      GoogleAdsClient googleAdsClient, long customerId, long merchantCenterAccountId) {
+
+    // Rejects a pending link request or unlinks an enabled link for a Google Ads account with
+    // customerId from a Merchant Center account with merchantCenterAccountId.
+    try (MerchantCenterLinkServiceClient merchantCenterLinkService =
+        googleAdsClient.getLatestVersion().createMerchantCenterLinkServiceClient()) {
+      ListMerchantCenterLinksResponse response =
+          merchantCenterLinkService.listMerchantCenterLinks(
+              ListMerchantCenterLinksRequest.newBuilder()
+                  .setCustomerId(Long.toString(customerId))
+                  .build());
+
+      System.out.printf(
+          "%d Merchant Center link(s) found with the following details:%n",
+          response.getMerchantCenterLinksCount());
+
+      for (MerchantCenterLink merchantCenterLink : response.getMerchantCenterLinksList()) {
+        System.out.printf(
+            "Link '%s' is in status '%s'.%n",
+            merchantCenterLink.getResourceName(), merchantCenterLink.getStatus());
+
+        // Checks if there is a link for the Merchant Center account we are looking for.
+        if (merchantCenterAccountId == merchantCenterLink.getId().getValue()) {
+          // If the Merchant Center link is pending, reject it by removing the link.
+          // If the Merchant Center link is enabled, unlink Merchant Center from Google Ads by
+          // removing the link.
+          // In both cases, the remove action is the same.
+          removeMerchantCenterLink(merchantCenterLinkService, customerId, merchantCenterLink);
+          // There is only one MerchantCenterLink object for a given Google Ads account and Merchant
+          // Center account, so we can break early.
+          break;
+        }
```

Comment:
Can we explicitly print when ID not found (something like below)? May be confusing for beginners who run this out of the box if it just prints out the list of links and nothing else.  
```suggestion
        } else {
          System.out.println("No Merchant Center link found for account ID %d.%n, merchantCenterAccountId);
        }
```

Reply:
Discussed with Adam offline. Concluded that it would complicate the example to do this correctly. We have chose to leave as is.